app/controllers/jobs_controller.rb

```diff
@@ -1,12 +1,27 @@
 class JobsController < ApplicationController
   def index
-    @company = Company.find(params[:company_id])
-    @jobs = @company.jobs
+    @jobs_by_city = Job.all.pluck(:city)
+    if params[:sort] == "interest"
+      @all_jobs = Job.order(level_of_interest: :desc)
```

Comment:
i suggest encapsulating the querying here in a model method rather than keeping it in your controller - this way you can reuse it if necessary